index.html

```diff
@@ -195,90 +203,98 @@ <h2 class="works-card-title">Multi-Post Stories</h2>
-            /></a>
-          </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/angellist.svg.svg" alt="github icon"
-            /></a>
+      <div class="about-me-description">
+        <div class="about-heading">
+          <h2 class="about-about-h2">About Myself</h2>
+          <p class="about-desc">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3 class="h3-conn">LET'S CONNECT</h3>
+          <ul class="about-icons">
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/github.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/twitter.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/angellist.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/linkedin.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/medium.svg.svg" alt="github icon"
+              /></a>
+            </li>
+          </ul>
+           <div class="about-btn">
+              <button type="button" class="btn">Get My Resume</button>
+            </div>
+        </div>
+      </div>
+      <div class="about-tech-stack">
+        <ul class="about-tools-list">
+          <li class="about-tool">
+            <div class="about-languages-list">
+              <h3 class="about-lang-title">Languages</h3>
+              <img
+                class="about-lang-img"
+                src="./assets/chev-down.png"
+                alt="arrow icon"
+              />
+            </div>
+            <ul class="about-lang">
+              <li class="about-lang-row">
+                <img src="./assets/javascript.svg.svg" alt="javascript icon" />
+                <p class="about-lang-name">JavaScript</p>
+              </li>
+              <li class="about-lang-row">
+                <img src="./assets/html.svg.svg" alt="javascript icon" />
+                <p class="about-lang-name">HTML</p>
+              </li>
+              <li class="about-lang-row">
+                <img src="./assets/css.svg.svg" alt="javascript icon" />
+                <p class="about-lang-name">CSS</p>
+              </li>
```

Comment:
- [ OPTIONAL]  Please refer to this [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)  to see how this section is aligned, consider resing the with as per the Figma, you can get a hint on the image bellow ⏬ 
- 
<img width="508" alt="lang" src="https://user-images.githubusercontent.com/79058364/155852195-5a74a3a0-8ebc-419b-a3cf-cac0365138f1.png">